var/spack/repos/builtin/packages/qt/package.py

```diff
@@ -70,15 +87,17 @@ class Qt(Package):
     # https://github.com/spack/spack/issues/1517
     patch('qt5-pcre.patch', when='@5:')
 
-    patch('qt4-corewlan-new-osx.patch', when='@4')
```

Comment:
It doesn't appear that this patch is used anymore but the patch file itself was not removed.

Out of curiosity did the application of this patch cause the install to fail, or did you find it wasn't required (unlike `qt4-el-capitan.patch` it doesn't appear that this was combined into another patch file)?

Reply:
Removing it now. It wasn't required (corewlan still appears to build), and the patch's changes would only affect Mac OS 10.8–13, and the current version is 10.15.